include/seqan3/search/configuration/all.hpp

```diff
@@ -57,14 +57,39 @@
  * | --------------------------------------------------------------------|-------|-------|-------|-------|-------|
  * | \ref seqan3::search_cfg::max_error  "0: Max error"                  |  ❌   |  ❌   |  ✅   |  ✅   |  ✅   |
  * | \ref seqan3::search_cfg::max_error_rate "1: Max error rate"         |  ❌   |  ❌   |  ✅   |  ✅   |  ✅   |
- * | \ref seqan3::search_cfg::output "2: Output"                         |  ✅   |  ✅   |  ❌   |  ✅   |  ✅   |
+ * | \ref search_configuration_subsection_output "2: Output"             |  ✅   |  ✅   |  ❌   |  ✅   |  ✅   |
  * | \ref search_configuration_subsection_hit_strategy "3. Hit"          |  ✅   |  ✅   |  ✅   |  ❌   |  ✅   |
  * | \ref seqan3::search_cfg::parallel "4: Parallel"                     |  ✅   |  ✅   |  ✅   |  ✅   |  ❌   |
  *
- * \subsection search_configuration_search_result Search result type
+ * \subsection search_configuration_subsection_output 2. Output Configuration
+ *
+ * The output configuration is closely tied to the seqan3::search_result:
  *
  * \copydetails seqan3::search_result
  *
+ * #### Configuring the result type
+ *
+ * As mentioned above, we can configure which information are accessible in the seqan3::search_result.
+ * For each member function there is a respective configuration element:
+ *
+ * * seqan3::search_cfg::output_query_id
+ * * seqan3::search_cfg::output_reference_id
+ * * seqan3::search_cfg::output_reference_begin_pos
+ * * seqan3::search_cfg::output_index_cursor
+ *
+ * If you specify any of the above mentioned output configuration elements, then nothing else but the selected
+ * output information is included.
+ *
+ * \include test/snippet/search/configuration_output.cpp
+ *
+ * The index cursor is an advanced data structure that let's you navigate within the index.
+ * See seqan3::fm_index_cursor and seqan3::bi_fm_index_cursor for more information.
+ * Note that only returning the cursor, without the reference id and positions is faster.
```

Comment:
In general the sentence is difficult to interpret. Because it only helps me if I don‘t need the positions. If I need the positions then I still need to locate on the cursor butwould need to do it later. So that is not faster.

Reply:
```suggestion
 * If you don't need the reference id or position, returning only the cursor is faster.
```